src/components/SearchByText/TextSearchBox.js

```diff
@@ -50,24 +53,25 @@ class TextSearchBox extends PureComponent /*:: <Props, State> */ {
     inputRef: T.func,
```

Comment:
InputRef can be removed as it is not used. This is what forwardRef is doing right?

Reply:
Thanks indeed, this is leftover from before I found the solution via ForwardRef